setup.cfg

```
@@ -86,5 +86,5 @@ exclude_lines =
     # Don't complain about IPython completion helper
     def _ipython_key_completions_
 
-[pep8]
+[pycodestyle]
```

```suggestion
[pep8]
```
This needs to stay as `[pep8]` for codeclimate; see [here](https://docs.codeclimate.com/docs/pep8)